DataBiosphere/Saturn-UI
# [PF-692] Phase 3 regionality release (support Montreal buckets)

Follow up from #2680. Phase 3 allows the UI to create workspace buckets in northamerica-northeast1 (Montreal). 

To test:
1) Create a new workspace. Verify that the location dropdown now includes Montreal. Create a workspace in Montreal.
2) Attempt to clone the new workspace. Verify the location dropdown includes US multi-regional, Montreal, and Iowa. Verify the egress warning appears properly when choosing a different location. Create a cloned workspace.
3) Repeat step 2 but for a workspace in the US. 

![image](https://user-images.githubusercontent.com/6879774/144649508-d025b9ba-46ed-4982-abb0-de2ac6cdd012.png)
